geode-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/server/AbstractRedisInfoStatsIntegrationTest.java

```diff
@@ -60,9 +61,20 @@
   private static final String NETWORK_KB_READ_OVER_LAST_SECOND = "instantaneous_input_kbps";
   private static final String UPTIME_IN_DAYS = "uptime_in_days";
   private static final String UPTIME_IN_SECONDS = "uptime_in_seconds";
+  private static final String USED_MEMORY = "used_memory";
+  private static final String MEMORY_FRAGMENTATION = "mem_fragmentation_ratio";
+  private static final String TCP_PORT = "tcp_port";
+  private static final String MAX_MEMORY = "maxmemory";
 
   private static final AtomicInteger numInfoCalled = new AtomicInteger(0);
 
+  public int getExposedPort() {
+    return -1; // This must be overridden by implementation classes
+  }
+
+  public void configureMaxMemory(Jedis jedis) throws Exception {
+    throw new Exception("configureMaxMemory must be overridden");
+  }
```

Comment:
Maybe we could declare `configureMaxMemory` abstract as well, like `getExposedPort`